api/lcp/server.py

```diff
@@ -188,14 +180,20 @@ def _create_partial_license(self, configuration, patron_id=None, license_start=N
                 }
             },
             'rights': {
-                'print': int(configuration.max_printable_pages),
-                'copy': int(configuration.max_copiable_pages)
+                'print':
+                    int(configuration.max_printable_pages)
+                    if configuration.max_printable_pages is not None
+                    else None,
```

Comment:
We just ran into a problem (https://jira.nypl.org/browse/SIMPLY-3004) where null values in JSON documents caused a parser to malfunction. For the sake of simplicity I'd prefer to omit `print` and `copy` altogether if the values here are `None`.

There's an additional problem which is the spec doesn't say whether a null value here means "use the default behavior of not limiting this feature at all" or whether it means "disable this feature entirely", the exact opposite. So let's avoid the ambiguity.

Reply:
Changed the code accordingly